packages/gatsby-source-wordpress/src/normalize.js

```diff
@@ -129,18 +129,20 @@ exports.liftRenderedField = entities =>
 const seedConstant = `b2012db8-fafc-5a03-915f-e6016ff32086`
 const typeNamespaces = {}
 exports.createGatsbyIds = entities =>
-  entities.map(e => {
-    let namespace
-    if (typeNamespaces[e.__type]) {
-      namespace = typeNamespaces[e.__type]
-    } else {
-      typeNamespaces[e.__type] = uuidv5(e.__type, seedConstant)
-      namespace = typeNamespaces[e.__type]
-    }
+  entities
+    .filter(e => e.wordpress_id) // Excluding entities without ID
```

Comment:
This is perfect! Could you make it though a standalone function so that in the future, when we do add explicit support for entities without IDs, we can add exceptions for them. Like we do here https://github.com/gatsbyjs/gatsby/blob/cea8b50dcd46544ccb720201fd9bf6a985ada3d1/packages/gatsby-source-wordpress/src/normalize.js#L62